spec/models/mixins/supports_feature_mixin_spec.rb

```diff
@@ -200,4 +201,40 @@ def initialize(options = {})
       expect(NukeablePost.new(:bribe => false).supports_nuke?).to be true
     end
   end
+
+  context "stub_supports" do
```

Comment:
It's not clear immediately from reading the test, that stub_supports is not actual code but a spec helper instead, so recommend the following.

```suggestion
  context "stub_supports (spec helper)" do
```

Alternatively, maybe this should just go onto a different spec file altogether.

Reply:
moving it.